charts/jenkins/CHANGELOG.md

```diff
@@ -12,6 +12,11 @@ Use the following links to reference issues, PRs, and commits prior to v2.6.0.
 The change log until v1.5.7 was auto-generated based on git commits.
 Those entries include a reference to the git commit to be able to get more details.
 
+## 4.3.17
+
+
+Add a .gitignore to ignore the generated files.
```

Comment:
no need to bump chart?

Reply:
GH actions believes the opposite:

>  ✖︎ jenkins => (version: "4.3.16", path: "charts/jenkins") > chart version not ok. Needs a version bump! 

No idea why

Reply:
because you added a trailing new line or something in
charts/jenkins/VALUES_SUMMARY.md

https://github.com/jenkinsci/helm-charts/pull/840/commits/4d70ff7b19731eaabd79f031edbdb9646ba83e4b